Example/CombineSample/README.md

```diff
@@ -0,0 +1,64 @@
+# Firebase & Combine Sample
+
+This sample demonstrates how to use Firebase's Combine APIs.
+
+## How to use
+
+### Set up a Firebase project
+
+1. Create a new Firebase project via the [Firebase console](https://console.firebase.google.com/)
+1. Enable the required Firebase services in the Firebase project you created in step 1
```

Comment:
Two 1's.

Reply:
Markdown renders this correctly. Happy to change this to a properly enumerated list if our style guide mandates it.